Snowplow/SnowplowUtils.m

```diff
@@ -103,6 +103,40 @@ + (NSString *) getCarrierName {
 #endif
 }
 
+
++ (NSString *) getNetworkType {
+#if TARGET_OS_IPHONE
+    Reachability *reachability = [Reachability reachabilityForInternetConnection];
+    [reachability startNotifier];
+    
+    NetworkStatus status = [reachability currentReachabilityStatus];
+    
+    if(status == NotReachable)
+    {
+        return @"none";
+    }
+    else if (status == ReachableViaWiFi)
+    {
+        return @"wifi";
+    }
+    else if (status == ReachableViaWWAN)
+    {
+        reutrn @"mobile";
```

Comment:
Note to self: Pushing commits before running off to Twitter events is poor form.